pkg/cache/application_test.go

```diff
@@ -177,6 +263,32 @@ func (ms *mockSchedulerAPI) ReloadConfiguration(rmID string) error {
 	return nil
 }
 
+type mockRecorder struct {
+	updateFn func()
```

Comment:
pls rename updateFn to `OnEventf()`, which means this is to be called when `Eventf()` is called

Reply:
sure